src/EnergyPlus/CMakeLists.txt

```diff
@@ -570,7 +570,12 @@ install( TARGETS energyplusapi
   LIBRARY DESTINATION ./ 
 )
 
-if (CMAKE_COMPILER_IS_GNUCXX AND ENABLE_LTO)
+if (MINGW)
```

Comment:
@jasondegraw sure if you have the time and interest.  I actually got LTO working by linking energyplus executable to the api, and got the performance bump too.  But then when I layered PGO back in things broke down again.  I don't know what is going on.  Its trying to inline things recursively.  I have a very loose grasp on what those optimizations do in the first place so I'm basically a monkey pulling levers and observing the results.